examples/ml-slurm-g4.yaml

```diff
@@ -0,0 +1,144 @@
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: ml-slurm-g4
+
+vars:
+  project_id: ## Set GCP Project ID Here ##
+  deployment_name: ## Set Deployment Name Here ##
+  region: ## Set GCP Region Here ##
+  zone: ## Set GCP Zone ID Here ##
+  net0_range: 192.168.0.0/19
+  filestore_ip_range: 192.168.32.0/24
+  net1_range: 192.168.64.0/18
+  new_image:
+    family: slurm-gcp-6-11-ubuntu-2204-lts-nvidia-570
+    project: schedmd-slurm-public
+    disk_size_gb: 100
+
+deployment_groups:
+- group: cluster-env
+  modules:
+  - id: net0
+    source: modules/network/vpc
+    settings:
+      network_name: $(vars.deployment_name)-net-0
+      mtu: 8896
+      subnetworks:
+      - subnet_name: $(vars.deployment_name)-sub-0
+        subnet_region: $(vars.region)
+        subnet_ip: $(vars.net0_range)
+       
+
+  - id: net1
+    source: modules/network/vpc
+    settings:
+      network_name: $(vars.deployment_name)-net-1
+      mtu: 8896
+      subnetworks:
+      - subnet_name: $(vars.deployment_name)-sub-1
+        subnet_region: $(vars.region)
+        subnet_ip: $(vars.net1_range)
+
+  - id: homefs
+    source: modules/file-system/filestore
+    use:
+    - net0
+    settings:
+      filestore_tier: HIGH_SCALE_SSD
+      size_gb: 10240
+      local_mount: /home
+      reserved_ip_range: $(vars.filestore_ip_range)
+      deletion_protection:
+        enabled: true
+        reason: Avoid data loss
+    outputs:
+    - network_storage
+  
+  # - id: private_service_access
+  #   source: community/modules/network/private-service-access
+  #   use: [net0]
+
+  # To use Managed Lustre as for the shared /home directory:
+  # 1. Comment out the filestore block above and the `filestore_ip_range` line in the vars block.
+  # 2. Uncomment the managed-lustre and private-service-access blocks.
+  # 3. Ensure the instance_image being used has the Lustre client installed.
+  # - id: homefs
+  #   source: modules/file-system/managed-lustre
+  #   use:
+  #   - net0
+  #   - private_service_access
+  #   settings:
+  #     size_gib: 18000
+  #     name: lustre-instance1
+  #     local_mount: /home
+  #     remote_mount: lustrefs
+  #     outputs:
+  #     - network_storage
+
+- group: cluster
+  modules:
+  - id: g4_nodeset
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    settings:
+      subnetwork_self_link: $(net0.subnetwork_self_link)
+      node_count_dynamic_max: 0
+      enable_placement: false
+      node_count_static: 1
+      bandwidth_tier: gvnic_enabled
+      machine_type: g4-standard-48
```

Comment:
if you use a g4-standard-48, you will only get one NIC, the only g4 that has two nics is the g4-standard-384 (https://cloud.google.com/compute/docs/gpus/gpu-network-bandwidth#g4-bandwidth). This should be at least documented in this blueprint, if not automated.